packages/frontend/src/components/wallet/NFTs.js

```diff
@@ -59,6 +59,7 @@ const StyledContainer = styled.div`
 
 const NFTs = ({ tokens }) => {
     if (tokens.length) {
+        tokens = tokens.filter(tokenDetails => tokenDetails.ownedTokensMetadata && tokenDetails.ownedTokensMetadata.length)
         return (
             <StyledContainer>
                 {tokens.map((tokenDetails) => (
```

Comment:
Thanks for making the change @june07gh! 🙏  Can we just move the filter down to `L64`?

Reply:
👍